assets/prototype/domain/eventEditor/containers/queries/useFetchRelations.js

```diff
@@ -1,40 +1,24 @@
 import { useQuery } from '@apollo/react-hooks';
 import { useEffect, useState } from '@wordpress/element';
 import { GET_RELATIONS_DATA } from './relations';
-import useToaster from '../../../../infrastructure/services/toaster/useToaster';
+import useInitToaster from '../../../../infrastructure/services/toaster/useInitToaster';
 
 const useFetchRelations = ({ eventId, onReceiveRelations }) => {
 	console.log('%c useFetchRelations: ', 'color: deeppink; font-size: 14px;');
-	const [initialized, setInitialized] = useState(false);
-
-	const toaster = useToaster();
 	const initializingMessage = 'initializing event editor relations';
 
+	const initToaster = useInitToaster({
+		toasterMessage: initializingMessage,
+		loadingMessage: initializingMessage,
+		successMessage: 'event editor relations initialized'
+	});
```

Comment:
can't think of any more lame jokes... but you get the idea... these all need updating